docs/installation.adoc

```diff
@@ -0,0 +1,142 @@
+= Installation Guide
+
+This document provides information on the installation of `odo` on various operating systems and the location of the binaries and the
+releases.
+
+* The latest `odo` binaries are available on
+https://dl.bintray.com/odo/odo/latest/[Bintray].
+* The latest `odo` releases are available on
+https://github.com/openshift/odo/releases/latest[Github].
+
+== Installing `odo`
+
+=== macOS
+
+. Access the repository using the Homebrew Tap as follows:
++
+[source,sh]
+----
+brew tap kadel/odo
+----
+
+. Install from the latest Master build or the latest released version:
+
+* For the latest Master build use:
++
+[source,sh]
+----
+brew install kadel/odo/odo --HEAD
+----
+* For the latest released version use:
++
+[source,sh]
+----
+brew install kadel/odo/odo
+----
+
+=== Ubuntu/Debian
+
+. Add the GPG key from the Bintray used to sign repositories:
++
+[source,sh]
+----
+curl -L https://bintray.com/user/downloadSubjectPublicKey?username=bintray | apt-key add -
+----
+
+. Add the repository to `/etc/apt/sources.list`:
+
+* For the latest Master builds use:
++
+[source,sh]
+----
+echo "deb https://dl.bintray.com/odo/odo-deb-dev stretch main" | sudo tee -a /etc/apt/sources.list
+----
+* For the latest signed releases use:
++
+[source,sh]
+----
+echo "deb https://dl.bintray.com/odo/odo-deb-releases stretch main" | sudo tee -a /etc/apt/sources.list
+----
+. Install odo:
++
+[source,sh]
+----
+sudo apt-get update
+sudo apt-get install odo
+----
+
+=== Fedora/CentOS/RHEL
+
+. Access the `odo` repository:
+
+* From the latest builds:
+.. Download the odo repository to `/etc/yum.repods.d/` using:
++
+[source,sh]
+----
+sudo curl -L https://bintray.com/odo/odo-rpm-dev/rpm -o /etc/yum.repos.d/bintray-odo-odo-rpm-dev.repo
+----
+.. Verify the content of the file and ensure that you see the following:
++
+....
+# cat/etc/yum.repos.d/bintray-odo-odo-rpm-dev.repo
+[bintraybintray-odo-odo-rpm-dev]
+name=bintray-odo-odo-rpm-dev
+baseurl=https://dl.bintray.com/odo/odo-rpm-dev
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+....
+* From the latest release:
+.. Download the odo repository to `/etc/yum.repods.d/` using:
++
+[source,sh]
+----
+sudo curl -L https://bintray.com/odo/odo-rpm-releases/rpm -o /etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
+----
+.. Verify the content of the file and ensure that you see the following:
++
+....
+# cat/etc/yum.repos.d/bintray-odo-odo-rpm-releases.repo
+[bintraybintray-odo-odo-rpm-releases]
+name=bintray-odo-odo-rpm-releases
+baseurl=https://dl.bintray.com/odo/odo-rpm-releases
+gpgcheck=0
+repo_gpgcheck=0
+enabled=1
+....
+. Install Odo:
+* For CentOS or RHEL use `yum install odo`.
+* For Fedora use `dnf install odo`.
```

Comment:
command on next line

Reply:
Fixed in the .adoc. we will be deleting the .md file, so not changing there.